doc/developer_guide.md

```diff
@@ -109,7 +109,7 @@ To use a local, non-published version of the extension interface for testing EM
 
 1. Build `extension-manager-interface` by running `npm run build`. This is required after each change.
 2. Edit [pkg/integrationTesting/extensionForTesting/package.json](./../pkg/integrationTesting/extensionForTesting/package.json) and replace the version of `"@exasol/extension-manager-interface"` with the path to your local clone of [extension-manager-interface](https://github.com/exasol/extension-manager-interface).
-3. Edit [pkg/integrationTesting/extensionForTesting/extensionForTestingTemplate.ts](./../pkg/integrationTesting/extensionForTesting/extensionForTestingTemplate.ts) and adapt it to the new API if necessary.
+3. Edit [pkg/integrationTesting/extensionForTesting/extensionForTestingTemplate.js](./../pkg/integrationTesting/extensionForTesting/extensionForTestingTemplate.js) and adapt it to the new API if necessary.
```

Comment:
Next sentence might be out-dated, as (still) mentioning ts: "It is not valid TypeScript, so it's normal that the editor complains about the invalid syntax."